source/get-started/download-and-install.txt

```diff
@@ -59,7 +59,6 @@ Download and Install
                 cmake ..                                \
                     -DCMAKE_BUILD_TYPE=Release          \
                     -DCMAKE_CXX_STANDARD=17                     \
-                    -DMONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX=OFF
```

Comment:
Also remove the trailing backslash in the preceeding line, or consider folding the command into a single line.